container.te

```diff
@@ -1033,3 +1033,33 @@ dontaudit container_domain device_node:chr_file setattr;
 dontaudit container_domain sysctl_type:file write;
 
 allow container_t proc_t:filesystem remount;
+
+
+########################################
+#
+# virt_launcher_t local policy
+#
+# SELinux security policy for kubevirt project
+# Temporary solution for rhbz#1795975
+
+type virt_launcher_t, container_domain;
+domain_type(virt_launcher_t)
+
+allow virt_launcher_t self:tun_socket { attach_queue relabelfrom relabelto create_socket_perms };
+
+manage_dirs_pattern(virt_launcher_t, container_var_run_t, container_var_run_t)
+manage_files_pattern(virt_launcher_t, container_var_run_t, container_var_run_t)
+
+manage_dirs_pattern(virt_launcher_t, container_share_t, container_share_t)
+manage_files_pattern(virt_launcher_t, container_share_t, container_share_t)
+
+manage_dirs_pattern(virt_launcher_t, container_runtime_tmp_t, container_runtime_tmp_t)
+manage_files_pattern(virt_launcher_t, container_runtime_tmp_t, container_runtime_tmp_t)
+files_tmp_filetrans(virt_launcher_t, container_runtime_tmpfs_t, { dir file })
+
+kernel_request_load_module(virt_launcher_t)
+
+dev_rw_mtrr(virt_launcher_t)
+dev_rw_sysfs(virt_launcher_t)
+
+virt_sandbox_net_domain(virt_launcher_t)
```

Comment:
I would prefer if you added the container_domain to this?

Reply:
if I understand what you meant by this, the parent (container_domain) of this is the domain that should be added to sandbox_net_domain?

Reply:
@rhatdan, We cannot assign attribute to attribute in m4 :/

Reply:
No I am suggesting that 
typeattribute virt_launcer_t, container_domain;

Reply:
Policy updated.